### ISSUE TITLE
Are the Parameter of the chainer version of inception model the same as the TF's ?

### ISSUE THREAD

**Issue by mathfinder** [pfnet-research/sngan_projection#26]
Are the Parameters in the inception model for inception score and FID the same as tensorflow's ?

**Comment by takerum** [comment]
Yes, but I found that the inception score by the Chainer code (https://github.com/pfnet-research/sngan_projection/blob/master/source/inception/inception_score.py) is slightly higher than the value by TF code. I am still not sure why.  
The reported values shown here are calculated with the original TF implementation (https://github.com/pfnet-research/sngan_projection/blob/master/source/inception/inception_score.py).

**Comment by mathfinder** [comment]
I think the code is correct.  I also found that the inception score and FID by the pytorch code is different from TF code. I think it is because the pytorch version inception model is trained by pytorch official, instead of migrating model parameters of the TF version. So I want to know the pretrain model is trained chainer official or migrated from TF pretrain model.

**Comment by takerum** [comment]
the pretrained model used in the chainer code is migrated from TF pretrained model.
please see this https://github.com/pfnet-research/sngan_projection/blob/master/source/inception/download.py
